src/app/routes/admin/ui/testpage/TestContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
```

Comment:
src/app/routes/admin/ui - it's not right place for components. Right place is  src/admin/ui